### DISCUSSION TITLE
Horizontal vs Vertical arrangement of slices

### DISCUSSION THREAD

**Discussion by joshka** [loco-rs/loco#876] [Ideas]
Vertical arrangement of slices of an application lead to being able to make changes to the parts of an application all at once without navigating all over the directory tree. This means collocating the models, views, and controllers in the same directory folders.

E.g. you might end up with something like:

```
src/posts/
- controller.rs
- view.rs
- api.rs
- model.rs
- views
  - add.html
  - get_one.html
  - list.html
...
```

Or similar. The obvious con is that you then end up with a lot of similarly named files.

**Comment by Sillyvan** [comment]
I personally do like vertical a lot but no hard feelings here. Changing it now might be a bit annoying for most users.

**Comment by snoweuph** [comment]
would be coll if this would just be a simple option
